internal/service/influxdb/resource_influxdb_database_test.go

```diff
@@ -127,28 +127,28 @@ func testAccInfluxDBDatabaseResource(project string, name string) string {
 		data "aiven_project" "foo" {
 		  project = "%s"
 		}
-
+		
```

Comment:
Hmm, it seem there are spaces on an empty line.

Reply:
Ok for now i'd say. Perspectively we should use https://github.com/katbyte/terrafmt and take this as ground truth then.

Reply:
OK, let's merged it for now